docs/primer/releases_and_records.md

```diff
@@ -14,22 +14,22 @@ Open Contracting Data Standard (OCDS) data is published and updated over time us
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/OeDCLMWlY_U" title="Introducing releases and records" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
-The only thing that can be truly called “OCDS data” is a JSON document that validates against the OCDS schema. The JSON documents that OCDS uses are called “releases” and “records.”
+The only thing that can be truly called “OCDS data” is JSON text that validates against the OCDS schema. The JSON texts that OCDS uses are called “releases” and “records.”
```

Comment:
I'm not sure about the use of 'JSON text'. Whilst, strictly speaking, it is the correct term to cover both JSON files and JSON served via an API, I think it could be confusing for less technical readers who might associate 'text' with unstructured data.

My feedback was simply to remove the bullet point about (non-JSON) documents from this page that is otherwise about releases and records, to which documents aren't relevant.

I guess we adopted 'JSON document' from https://json-schema.org/. I don't remember it causing any confusion previously so I'm not sure we need to change it.

If we do need to change 'JSON document', I think 'JSON data' would be a better choice. For example, this paragraph could read:

> The only thing that can be truly called “OCDS data” is JSON data that validates against the OCDS schema. There are two schemas for OCDS data: “releases” and “records.”

Reply:
Looks good!